pandas/tests/extension/base/methods.py

```diff
@@ -421,3 +421,29 @@ def test_repeat_raises(self, data, repeats, kwargs, error, msg, use_numpy):
                 np.repeat(data, repeats, **kwargs)
             else:
                 data.repeat(repeats, **kwargs)
+
+    def test_equals(self, data, na_value, as_series):
```

Comment:
Can you also include `DataFrame` in this parameterization, or do it somewhere else? Just to ensure coverage of the changes to `blocks.py`.